### ISSUE TITLE
Specify which permissions the GITHUB_TOKEN needs to include

### ISSUE THREAD

**Issue by simenbrekken** [binaryberry/seal#147]
Under the ["How to use it"](https://github.com/binaryberry/seal#how-to-use-it) section of the README, it doesn't specify which permissions the Github token needs to have.

**Comment by KamuelaFranco** [comment]
I agree. Also, GitHub's descriptions aren't necessarily intuitive which compounds this issue. Can anyone provide some insight?

**Comment by kentcb** [comment]
Yep, just burnt > 1 hour on this because my repos were private.

**Comment by ericjorgensen** [comment]
@kentcb What permissions did you need to set to make this work for private repositories?

**Comment by bbruneau** [comment]
@kentcb can you share which?
